tracer/src/Datadog.Trace.Tools.Runner/RunCiCommand.cs

```diff
@@ -366,8 +366,18 @@ private async Task ExecuteAsync(InvocationContext context)
                                     // Found using the OpenCover format.
 
                                     // Adds the global code coverage percentage to the session
+                                    var coveragePercentage = Math.Round(seqCovValue, 2);
+                                    if (double.IsNaN(coveragePercentage) || double.IsNegativeInfinity(coveragePercentage))
```

Comment:
Should we be catching _any_ negative number here, not just negative infinity? 🤔

Reply:
true, done in [710e28d](https://github.com/DataDog/dd-trace-dotnet/pull/4661/commits/710e28dfb58981be2695c65bb17bb884400410cc)